types/react-input-mask/v2/index.d.ts

```diff
@@ -0,0 +1,75 @@
+// Type definitions for react-input-mask 2.0.3
+// Project: https://github.com/sanniassin/react-input-mask
+// Definitions by: Alexandre Paré <https://github.com/apare>
+//                 Dima Danylyuk <https://github.com/dima7a14>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.9.2
+
+declare module 'react-input-mask' {
```

Comment:
The module declaration seems can be removed

Reply:
Thank you. But without module declaration I have ts error: *Try `npm install @types/react-input-mask` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-input-mask';`*.

Reply:
The module declaration is used when the definition is not in `react-input-mask` package and `@types/react-input-mask` package, your code will be in `@types/react-input-mask`, so not needed.
You can check other packages' types

Reply:
You are right, thanks.